executor/distsql_test.go

```diff
@@ -316,6 +316,9 @@ func TestPartitionTableIndexLookUpReader(t *testing.T) {
 	tk.MustQuery("select * from t where a>=1 and a<15 order by a").Check(testkit.Rows("1 1", "2 2", "11 11", "12 12"))
 	tk.MustQuery("select * from t where a>=1 and a<15 order by a limit 1").Check(testkit.Rows("1 1"))
 	tk.MustQuery("select * from t where a>=1 and a<15 order by a limit 3").Check(testkit.Rows("1 1", "2 2", "11 11"))
+	tk.MustQuery("select * from t where a>=1 and a<15 limit 3").Check(testkit.Rows("1 1", "2 2", "11 11"))
+	tk.MustQuery("select * from t where a between 1 and 15 limit 3").Check(testkit.Rows("1 1", "2 2", "11 11"))
+	tk.MustQuery("select * from t where a between 1 and 15 limit 3 offset 1").Check(testkit.Rows("2 2", "11 11", "12 12"))
```

Comment:
Will IndexLookupReader always sort the rows, or can these test be instable, due to the rows can be out-of-order or even that "12 12" could be returned instead of "1 1" or "2 2" due to the partitions may be read in parallel?

Reply:
IndexWorker not work in parallel to deal with the partition table right now. So it always works well.
https://github.com/pingcap/tidb/pull/32548/files#diff-d73d3a1acc55230180db0c3b4fbd89bead84e15845062549e20ec9e810a283b9L591